website/pages/docs/plugins/destinations/snowflake/overview.md

```diff
@@ -38,7 +38,7 @@ This is the top level spec used by the Snowflake destination plugin.
 
   Example `connection_string`:
 
-  ```
+  ```bash
```

Comment:
Should this be bash? I don't know what it should be, but bash doesn't use `//` for comments 🙂

Reply:
Good point, change it to `yaml` as it's part of a spec example